game/environment.go

```diff
@@ -553,16 +713,16 @@ func (g *environment) configureMonitor(bridge string, numberNetworks int) error
 
 	}
 
-	macAddress := g.dhcp.GetMAC()
+	macAddress := env.dhcp.GetMAC()
 	macAddressClean := strings.ReplaceAll(macAddress, ":", "")
 	nicNumber := len(ifaces) + 1
 
 	fmt.Println(macAddressClean)
 	fmt.Println(nicNumber)
 	//
 	fmt.Println(ifaces)
```

Comment:
Debug stuff should be deleted before merge, you can also use the log.Debug().Str("mac",macAddressClean).Msg("configuring SOC")